### PR TITLE
Move securefile-common to shared package

### PR DESCRIPTION
**Pull request by damccorm**
**Description**: Move securefiles-common to shared npm package instead of common folder. [Source code](https://github.com/microsoft/azure-pipelines-tasks-common/tree/main/securefiles-common)

**Documentation changes required:** N

**Added unit tests:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/Common/kubernetes-common-v2/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -2,5 +2,6 @@
   "loc.messages.UnableToFetchKubectlVersion": "Could not fetch Kubectl version. Please make sure that the Kubernetes server is up and running.",
   "loc.messages.KubectlClientVersion": "Kubectl Client Version",
   "loc.messages.KubectlServerVersion": "Kubectl Server Version",
-  "loc.messages.KubectlServerVerisonNotFound": "Could not find kubectl server version"
+  "loc.messages.KubectlServerVerisonNotFound": "Could not find kubectl server version",
```
Comment (stephenmichaelf) [inline Tasks/Common/kubernetes-common-v2/Strings/resources.resjson/en-US/resources.resjson:5]: Is this file supposed to be changed?

Comment (damccorm) [reply Tasks/Common/kubernetes-common-v2/Strings/resources.resjson/en-US/resources.resjson]: Nope, undid that change!

Comment (damccorm) [reply Tasks/Common/kubernetes-common-v2/Strings/resources.resjson/en-US/resources.resjson]: (that's what was causing CI to fail 😅)

File: Tasks/InstallAppleCertificateV2/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Install Apple certificate",
-  "loc.helpMarkDown": "[More information](https://go.microsoft.com/fwlink/?LinkID=862067)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=862067)",
```
Comment (stephenmichaelf) [inline Tasks/InstallAppleCertificateV2/Strings/resources.resjson/en-US/resources.resjson:3]: Is this an intentional change?

File: Tasks/InstallAppleProvisioningProfileV1/Strings/resources.resjson/en-US/resources.resjson [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Install Apple provisioning profile",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=862068)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=862068)",
```
Comment (stephenmichaelf) [inline Tasks/InstallAppleProvisioningProfileV1/Strings/resources.resjson/en-US/resources.resjson:3]: Are these intentional changes?

File: Tasks/DownloadSecureFileV1/Strings/resources.resjson/en-US/resources.resjson [hunk 4]
```diff
@@ -7,4 +7,4 @@
   "loc.input.help.secureFile": "The file name or GUID of the secure file to download to the agent machine. The file will be deleted after the pipeline runs.",
   "loc.input.label.retryCount": "Retry Count",
   "loc.input.help.retryCount": "Optional number of times to retry downloading a secure file if the download fails."
-}
+}
```
Comment (stephenmichaelf) [inline Tasks/DownloadSecureFileV1/Strings/resources.resjson/en-US/resources.resjson:10]: ?

File: Tasks/InstallAppleCertificateV2/Strings/resources.resjson/en-US/resources.resjson (hunk 2 above)
Comment (damccorm) [reply Tasks/InstallAppleCertificateV2/Strings/resources.resjson/en-US/resources.resjson:3]: This is done automatically when we build, its from the localization I guess?

File: Tasks/InstallAppleProvisioningProfileV1/Strings/resources.resjson/en-US/resources.resjson (hunk 3 above)
Comment (damccorm) [reply Tasks/InstallAppleProvisioningProfileV1/Strings/resources.resjson/en-US/resources.resjson:3]: Same as above - autogenerated by the build

File: Tasks/DownloadSecureFileV1/Strings/resources.resjson/en-US/resources.resjson (hunk 4 above)
Comment (damccorm) [reply Tasks/DownloadSecureFileV1/Strings/resources.resjson/en-US/resources.resjson:10]: Same as above - autogenerated by the build/maybe git, I think someone probably had bad line endings when they checked this one in

**Pull request merged**